composer.json

```diff
@@ -14,7 +14,8 @@
     "justinrainbow/json-schema": "^1.6 || ^2.0 || ^4.0",
     "lastguest/murmurhash": "1.3.0",
     "guzzlehttp/guzzle": "~5.3|~6.2",
-    "monolog/monolog": "~1.21"
+    "monolog/monolog": "~1.21",
+    "ramsey/uuid": "^3.7"
```

Comment:
why did u add this module?

Reply:
To generate random UUID

Reply:
can we use this one method? uuid_create?
I don't think so, we need to add any lilbrary to create guid?

Reply:
Copy guid generated method from stackoverflow.